### ISSUE TITLE
When using Multipass, closing or opening any server transport will disconnect the local client.

### ISSUE THREAD

**Issue by Disastercake** [FirstGearGames/FishNet#351]
Unity version: 2021.3.22
Fish-Networking version: 3.6.7.Pro

**Describe the bug**
When using Multipass, closing or opening any server transport will disconnect the local client.

**To Reproduce**
1. Create a project with individual toggles for each transport using Multipass.
2. Connect the local client to one of the transports (like Yak).
3. Turn on or off a transport that is not Yak (like Tugboat or FishySteamworks).
4. See that the local client disconnects.

**Expected behavior**
Every client should remain unaffected by a transport opening or closing, unless they are actually connected through that transport.

**Comment by FirstGearGames** [comment]
I assume clientHost is getting dropped not other clients?

**Comment by FirstGearGames** [comment]
Resolved in 3.7.10
Added this check in `ClientObjects.cs` OnServerConnectionState.
```cs
            /* Only perform this step if the transport being stopped
             * is the one which client is connected to. */
            if (NetworkManager.IsClient && args.TransportIndex == base.NetworkManager.ClientManager.GetTransportIndex())
                base.NetworkManager.ClientManager.StopConnection();
```
